src/Core/Services/Implementations/HandlebarsMailService.cs

```diff
@@ -378,6 +378,21 @@ public async Task SendLicenseExpiredAsync(IEnumerable<string> emails, string org
         await _mailDeliveryService.SendEmailAsync(message);
     }
 
+    public async Task SendRequestSMAccessToAdminEmailAsync(IEnumerable<string> emails, string organizationName, string requestingUserName, string emailContent)
+    {
+        var message = CreateDefaultMessage("Access Requested for Secrets Manager", emails);
+        var model = new RequestSecretsManagerAccessViewModel
+        {
+            OrgName = CoreHelpers.SanitizeForEmail(organizationName, false),
+            UserNameRequestingAccess = CoreHelpers.SanitizeForEmail(requestingUserName, false),
+            EmailContent = CoreHelpers.SanitizeForEmail(emailContent),
+            TrySecretsManagerUrl = "https://bitwarden.com/contact-sales/"
```

Comment:
In some other templates, the URL is in the template. 

For example https://github.com/bitwarden/server/blob/fa4dc4aaf2f6b47cd4c6adc39ada24a0d6195bf3/src/Core/MailTemplates/Handlebars/OrganizationSmSeatsMaxReached.html.hbs#L18-L20

If we aren't constructing the link from data, it might be better to keep it as an email template concern?